fe/fe-core/src/test/java/com/starrocks/pseudocluster/PseudoBackend.java

```diff
@@ -831,6 +862,11 @@ void execBatchPlanFragment(TExecPlanFragmentParams commonParams, TExecPlanFragme
         report.setDone(true);
         // TODO: support error injection
         report.setStatus(new TStatus(TStatusCode.OK));
+        for (TPlanNode planNode : commonParams.fragment.plan.nodes) {
+            if (planNode.node_type == TPlanNodeType.OLAP_SCAN_NODE) {
```

Comment:
Does the new pipeline use another ScanOperator, not the OLAP_SCAN_NODE?

Reply:
I tested it, it works fine, so it should be OLAP_SCAN_NODE

Reply:
Maybe in the FE still use the OLAP_SCAN_NODE?
In BE, it have changed to exec/pipeline/scan/olap_scan_operator.cpp.

Reply:
That doesn't matter I think